code/nrf-connect/prstlib/boards/arm/bparasite_nrf52833/board.cmake

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nRF52833_xxAA" "--speed=4000")
+board_runner_args(pyocd "--target=nrf52833" "--frequency=4000000")
+# set(OPENOCD_NRF5_SUBFAMILY "nrf52")
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd-nrf5.board.cmake)
```

Comment:
Do you mind also adding

```
include(${ZEPHYR_BASE}/boards/common/blackmagicprobe.board.cmake)
```

This allows flashing and gdb step-through debugging through the blackmagic probe.

Reply:
Great idea, tks. Done.